### DISCUSSION TITLE
Can't clone src folder

### DISCUSSION THREAD

**Discussion by chmcquee** [InseeFrLab/onyxia#461] [Q&A]
Hi all,
I'm trying to get this demo project to run in order to understand how to customise components with keycloakify.
I attempted a clone but got the following error `git-lfs filter-process: git-lfs: command not found
fatal: the remote end hung up unexpectedly`
Thus the public folder is empty and the src folder doesn't even exist.
I'm grateful for any help you can offer.
Chris

**Comment by olevitt** [comment]
Hi !  
The error message appears to indicate an issue with your `git-lfs` local installation. It seems you don't have `git-lfs` installed or not correctly linked. Note that `git-lfs` is an extension of gît that may not be installed by default on your machine.  
See here for more détails : https://stackoverflow.com/questions/67395259/git-clone-git-lfs-filter-process-git-lfs-command-not-found

**Comment by garronej** [comment]
Hello @chmcquee,  

Running Onyxia for experimenting with Keycloakify seems a bit extreme but hey, why not 😄  
Here you have the technical documentation and a quick start for running this project locally: https://docs.onyxia.sh/contributing/onyxia/quick_start  

Otherwise you have the two project starters:  
https://github.com/garronej/keycloakify-starter  
https://github.com/garronej/keycloakify-advanced-starter  

Best,
